### ISSUE TITLE
Bug: Media file picker not sending audio file as audio

### ISSUE THREAD

**Issue by TheTrueDonDotta** [rhunk/SnapEnhance#983]
### Description

When i send an audio file through the media file picker feature it doesn't send as a voice note but as a video or actual media instead. Both options do this.
 
![Screenshot_20240430_210948_Snapchat](https://github.com/rhunk/SnapEnhance/assets/144651375/8bdab05c-9a99-49fd-8f59-a31663ef94ac)

### Reproduction steps

1. Turn on media file picker 
2. Go in chat and open media file picker
3. Send a audio file through media file picker
4. See error.

### Snapchat Version

12.84.0.40

### SnapEnhance Version

2.0.2

### Agreement

- [X] This is not a bug regarding Snapchat+.
- [X] I have provided a detailed description of the issue.
- [X] I have attached a log if deemed necessary.
- [X] This issue is not a duplicate.

**Comment by bocajthomas** [comment]
@TheTrueDonDotta I think that's what it's designed to do

**Comment by TheTrueDonDotta** [comment]
> @TheTrueDonDotta I think that's what it's designed to do 

Before when i would use the feature it would send audio files as a voice note so i dont think so.

**Comment by bocajthomas** [comment]
> Before when i would use the feature it would send audio files as a voice note so i dont think so. also even when i send a video and select audio only it doesnt work.

![Screenshot_20240430_214244_Snapchat](https://github.com/rhunk/SnapEnhance/assets/41988041/65287422-163b-4a2f-983a-2ef08c2ccb08)

Does this option appear after It's done converting?

**Comment by TheTrueDonDotta** [comment]
> > Before when i would use the feature it would send audio files as a voice note so i dont think so. also even when i send a video and select audio only it doesnt work.
> 
> ![Screenshot_20240430_214244_Snapchat](https://github.com/rhunk/SnapEnhance/assets/41988041/65287422-163b-4a2f-983a-2ef08c2ccb08)
> 
> Does this option appear after It's done converting?
> 
> 

yes it does. even when i turn off gallery override it still doesn't work tho.

**Comment by bocajthomas** [comment]
> yes it does. even when i turn off gallery override it still doesn't work tho.

I think this is fixed in v2.1.0 you will need to wait for a ci or stable
